SNIPS/snip-XXX.md

```diff
@@ -0,0 +1,111 @@
+
+> precision on the vote mechanism to be added & precise definition of "significant breaking change needed" (IMO any breaking change is significant)
+
+
+
+### **Breaking Change Timeline**
+
+The timeline proposed here reflects a minimal time to allow all relevant parties to learn about the breaking change and prepare for it. It is possible that some SNIPs will take longer to reach acceptance and deployment.
+
+- **T - 5 months:** The initial SNIP idea is discussed with relevant parties (e.g., Full Nodes, Wallets, StarkWare, etc.).
+  - A discussion is opened on the Starknet Community Forum.
+- **T - 4 months:** An initial SNIP draft is open as a PR to the Starknet-io SNIP repository.
+- **T - 3.5 months:**
+  - Initial comments on the PR are addressed and it is merged into the SNIP repository.
+- **T - 2.5 months:**
+  - Comments in the community forum are addressed, and there is a temperature gauge for whether the SNIP is accepted by the community.
+- **T - 2 months:**
+  - The SNIP is discussed in a Community Call hosted by the Starknet Foundation, where the author can discuss more details of the SNIP and its benefits. If there are no objections, the SNIP is en route to being introduced.
+  - The content of the community call is summarized in a public Github repository, with decisions regarding the SNIP.
+  - The SNIP is officially on the roadmap to be introduced in a Starknet version upgrade.
+  - A page summarizing the changes and the effects on relevant parties is added to a SNIPs page on Starknet.io.
+- **T - 1.5 months:**
+  - A mailing list is sent announcing the official date when the SNIP will be introduced to Starknet testnet/integration.
+  - The mailing list will include relevant links to the SNIP such as:
+    - Link to GH discussions.
+    - Link to the community forum.
+    - Link to the relevant community call.
+- **T - 2 weeks:** Additional reminder is sent with SNIP main bullet points and dates.
+- **T:** The SNIP is introduced to Starknet integration.
+- **T + ~1 month:** Provided no major issues, the SNIP is live on Starknet testnet.
+- **T + ~2 months:** The SNIP is live on Starknet mainnet.
+
+### **Fast track for Non-Breaking Changes**
+
+As the Starknet network is still in a growing phase a "fast track" is needed for more agility. Thus non-protocol-level SNIPs and SNIPs that do not introduce breaking changes can be "fast-tracked" but **SHOULD** still undergo a regular process to ensure sufficient coverage and comments from relevant community members. Fast-tracked EIPs can go from `draft` to `last call` directly per editor request. However as the network matures more decentralisation will be required and the "fast-track" **MUST** end in 87,600 blocks. 
```

Comment:
@Eikix As discussed, block time variates so it's not the best indicator, should I remove the block target ?

Reply:
I think you can add that until SN decentralizes, it is instead a date and time. When Starknet decentralizes it'll be a block target.